docs/getting_started.md

```diff
@@ -0,0 +1,262 @@
+To learn more about cluster API in more depth, check out the the [cluster api docs page](https://cluster-api.sigs.k8s.io/).
+
+Table of Contents
+=================
+
+   * [Getting Started](#getting-started)
+   * [Table of Contents](#table-of-contents)
+      * [Bootstrapping a Management Cluster with clusterctl](#bootstrapping-a-management-cluster-with-clusterctl)
+         * [Install Requirements](#install-requirements)
+            * [Kind](#kind)
+            * [envsubst](#envsubst)
+            * [clusterctl](#clusterctl)
+         * [vSphere Requirements](#vsphere-requirements)
+            * [vCenter Credentials](#vcenter-credentials)
+            * [Networking Requirements](#networking-requirements)
+         * [Generating YAML for the Bootstrap Cluster](#generating-yaml-for-the-bootstrap-cluster)
+         * [Using clusterctl](#using-clusterctl)
+         * [Managing Workload Clusters using the Management Cluster](#managing-workload-clusters-using-the-management-cluster)
+
+## Bootstrapping a Management Cluster with clusterctl
+
+`clusterctl` is a command line tool used for [bootstrapping](https://github.com/kubernetes-sigs/cluster-api/blob/master/docs/book/GLOSSARY.md#bootstrap) your [Mangement Cluster](https://github.com/kubernetes-sigs/cluster-api/blob/master/docs/book/GLOSSARY.md#management-cluster).
+Your management cluster stores resources such as `clusters` and `machines` using the Kubernetes API. This is the cluster you use to provision and manange multiple clusters going forward.
+Before diving into the bootstrap process, it's worth noting that a Management Cluster is just another Kubernetes cluster with special addons and CRDs. You are not required to use `clusterctl` to
+provision your management cluster, however, this guide will be focused on using `clusterctl` to boostrap your management cluster using [Kind](https://github.com/kubernetes-sigs/kind).
+
+### Install Requirements
+
+#### Kind
+
+Assuming you have a Go installed on your machine, run the following to install Kind:
+
+```bash
+$ GO111MODULE="on" go get sigs.k8s.io/kind@v0.3.0
+```
+
+#### envsubst
+
+envsubst is required for YAML template generation (more on this below).
+
+On **Linux** you can install envsubst by installing the gettext RPM.
+```
+$ apt install gettext
+# or...
+$ yum install gettext
+```
+
+On **MacOS** you can install envsubst with brew:
+```
+$ brew install gettext
+```
+
+NOTE: brew may install envsubst in a path that is not in your PATH environment variable. Ensure
+
+#### clusterctl
+
+A version of `clusterctl` that is built from **this** respository must be installed. If you have a workin Go environment,
+you can use the `clusterctl` make target:
+```
+$ make clusterctl
+```
+
+The `clusterctl` make target installs `clusterctls` in `bin/clusterctl`. Temporarily add this folder to your PATH to use it
+for the rest of this guide:
+```
+$ export PATH=$PATH:$(pwd)/bin
```

Comment:
Hi @andrewsykim,

Perhaps consider quoting it `export PATH="${PATH}:$(pwd)/bin"`